heaps/heap_sort.py

```diff
@@ -1,8 +1,18 @@
+from heaps.min_heap import MinHeap
 
 
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  On
+        Space Complexity: On
```

Comment:
✨ Time complexity here will be O(n log n) where n is length of `list`. For each of n items in `list`, you perform `heap.add()`/`heap.remove()` which are both log n operations.